ads/google/a4a/docs/RTB-Implementation.md

```diff
@@ -0,0 +1,65 @@
+* [AMP Ads Website from the AMP Project](http://www.ampproject.org/ads&sa=D&ust=1496436943732000&usg=AFQjCNEwQwnBRfEKRN6gftjeoPa_OYrlkQ)
+* [Example AMP Ads](https://ampbyexample.com/amp-ads/%23amp-ads/introduction&sa=D&ust=1496436943733000&usg=AFQjCNFwdvQ4G4V0TeBjKd7vfd1xohQQOQ)
+* [Speed comparison](https://ampbyexample.com/amp-ads/introduction/amp_ads_vs_non-amp_ads/&sa=D&ust=1496436943734000&usg=AFQjCNG4RdQVhjlh7Sbk9d5ylDptM4-Ffw): see how fast an AMP Ad loads in comparison to a regular ad. Best viewed on a 3G connection.
+* [Discussion in OpenRTB Dev Forum](https://groups.google.com/forum/%23!topic/openrtb-dev/0wyPsF5D07Q&sa=D&ust=1496436943736000&usg=AFQjCNFlwyrmUfG8v4VLgAuZUqSCroDAZw): RTB Specific Proposal
+ 
+## AMP Ads in RTB: High-Level Approach
+ 
+### RTB Bid Request
+ 
+Exchanges will need to indicate in the RTB bid request whether a page is built in AMP HTML, and any specific requirements or treatment of AMP Ads.  As of [OpenRTB 2.5](https://www.google.com/url?q=http://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf&sa=D&ust=1496436943740000&usg=AFQjCNGuz1b-E6kOzHQr9RJ6Xb_DNAy_xQ), this is not yet included in the spec, but the proposed implementation to the IAB committee is as follows.
+ 
+**Site Object Additional field: amp**
+A new field is added to the 'Site' object of the OpenRTB standard to indicate whether a webpage is built on AMP HTML.  In OpenRTB 2.5, this is section 3.2.13.
+ 
+![RTB1](./img1.png)
+ 
+**Imp Object Additional field: ampad**
+A new field is added to the 'Imp' object of the OpenRTB standard to provide more detail around AMP ad requirements and how AMP ads will load.  In OpenRTB 2.5, this is section 3.2.4.
+ 
+![RTB2](./img2.png)
+ 
+**AMP Ad Status Table**
+ 
+![RTB3](./img3.png)
+ 
+### RTB Bid Response
+ 
+SSPs will need to provide a new field in the bid response to allow bidders to return AMP Ad content, and RTB bidders will need to populate that field in order to return AMP Ads.  As of [OpenRTB 2.5](http://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf&sa=D&ust=1496436943812000&usg=AFQjCNEQipmPbKkfCuGkKvzgdpC-I60EOg), this is not yet included in the spec, but the proposed workflow is a new field that accepts a URL pointing to AMP ad content.  
+ 
+**AMP Ad Status Table**
+ 
+![RTB4](./img4.png)
+ 
+### Verification of valid AMP
+ 
+* For AMP Ads to be early rendered, the exchange is required to verify and sign in real time that the ad is written in amp4ads  <html amp4ads> creative format.
+* See "[Proposed Design](https://github.com/ampproject/amphtml/issues/3133&sa=D&ust=1496436943832000&usg=AFQjCNGwYUpiQAwBoxZLaBDfmDYurYeTWw)" for signing.
+* Ads that are valid AMP Ads will be allowed to render early by AMP pages.  Ads that are not verified as valid AMP Ads  will render at the same speed as non-AMP ads.
+* Only AMP Ads  should be returned in the ampadurl .
```

Comment:
This is a field, please use code format `ampadurl`.  Also remove space before period.